helix-view/src/document.rs

```diff
@@ -1036,6 +1036,12 @@ impl Document {
             .map(helix_core::path::get_relative_path)
     }
 
+    pub fn display_name(&self) -> String {
+        self.relative_path()
+            .map(|path| path.to_string_lossy().to_string())
```

Comment:
Rather than always allocating could this just return a Cow<>? Then the parent code can call to_owned() if it needs to